.github/workflows/webviz-subsurface-components.yml

```diff
@@ -19,7 +19,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9']
+        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        include:
+          - run-performance-tests: true
```

Comment:
It looks like this ends up with the following matrix:
`["3.6", true], ["3.7", true], ["3.8", true], ["3.9", true]`

While we want probably only one to be `true`. I can help research the GitHub workflow `matrix` syntax documentation a bit further.

Reply:
One way that should work would be to e.g.:
```
matrix:
  include:
    - python-version: 3.6
       run-performance-tests: false
    - python-version: 3.7
       run-performance-tests: false
    - python-version: 3.8
       run-performance-tests: false
    - python-version: 3.9
       run-performance-tests: false
    - python-version: 3.9
       run-performance-tests: false
```
The only downside would be that it is a bit more verbose, but maybe the best solution? See also https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#using-environment-variables-in-a-matrix. What do you think @avanti-em?